packages/account/src/Types/common.type.ts

```diff
@@ -162,3 +148,16 @@ export type TServerError = {
     details?: { [key: string]: string };
     fields?: string[];
 };
+
+export type TFinancialInformationForm = Pick<
+    SetFinancialAssessmentRequest,
+    | 'employment_status'
+    | 'income_source'
+    | 'employment_industry'
+    | 'occupation'
+    | 'source_of_wealth'
+    | 'education_level'
+    | 'net_income'
+    | 'estimated_worth'
+    | 'account_turnover'
+>;
```

Comment:
```suggestion
export type TFinancialInformationForm = Omit<SetFinancialAssessmentRequest, 'set_financial_assessment'>;
```

Reply:
The structure of `SetFinancialAssessmentRequest` is old and it contains Trading assessment form fields as well. So its smaller  to pick properties rather than omiting

Reply:
![Screenshot 2023-11-17 at 2 20 44 PM](https://github.com/binary-com/deriv-app/assets/129842155/5607c7ed-fc97-4f5b-949e-e477e1b7a96c)
We're getting this type error which I believe needs to be resolved & it's related to this defined type

Reply:
@shahzaib-deriv I have answered this before, but since u've commented below and I cannot reply there, I'm giving another try in this thread. The payload struture I'm looking for is not the whole `SetFinancialAssessmentRequest`.  I just need `9` property as part of Financial assessment. So I cannot use `Omit` The request structure as per me for FinancialAssessment and Trading assessment have to be updated from BE side. May be @amina-deriv can provide her inputs
<img width="1012" alt="Screenshot 2023-11-20 at 10 39 34 AM" src="https://github.com/binary-com/deriv-app/assets/98398322/7dc8635b-cf28-4b14-ae0e-588c631ecfe8">